app/views/directives/edit-environment-from.html

```diff
@@ -78,4 +84,44 @@
         ng-click="$ctrl.onAddRow()">{{ $ctrl.addRowLink }}</a>
     </div>
   </div>
+
+  <overlay-panel class="add-config-to-application" show-panel="$ctrl.overlayPanelVisible" show-close="true" handle-close="$ctrl.closeOverlayPanel">
+    <div class="dialog-title">
+      <h3>Value Details</h3>
+    </div>
+    <div class="modal-body">
+      <h4>{{$ctrl.overlayPaneEntryDetails.metadata.name}}
+        <small class="muted">- {{$ctrl.overlayPaneEntryDetails.kind | humanizeKind : true}}</small></h4>
+
+      <div ng-if="!($ctrl.overlayPaneEntryDetails.data | size)" class="empty-state-message text-center">
+        The {{$ctrl.overlayPaneEntryDetails.kind | humanizeKind : true}} has no properties.
```

Comment:
We use lowercase names for kinds in sentences throughout the console, so remove the `true` flag here.